src/main/java/seedu/address/logic/parser/ClientCliSyntax.java

```diff
@@ -0,0 +1,13 @@
+package seedu.address.logic.parser;
+
+/**
+ * Represents Prefix definitions for client commands.
+ */
+public class ClientCliSyntax {
+    /* Prefix definitions */
+    public static final Prefix PREFIX_CLIENT_NAME = new Prefix("n/");
+    public static final Prefix PREFIX_CLIENT_PHONE = new Prefix("p/");
+    public static final Prefix PREFIX_CLIENT_EMAIL = new Prefix("e/");
+    public static final Prefix PREFIX_PROJECT_ID = new Prefix("i/");
```

Comment:
not sure if you wanna standardise with issue, which uses proj/ for PROJECT_ID we can discuss this